src/main/java/seedu/address/logic/commands/DoneCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.results.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.item.Note;
+
+/**
+ * This command allows user to mark a note as done.
+ */
+public class DoneCommand extends Command {
+    public static final String COMMAND_WORD = "done";
+
+    public static final String MESSAGE_DONE_SUCCESS = "Mark this note as done!";
```

Comment:
```suggestion
    public static final String MESSAGE_DONE_SUCCESS = "Marked this note as done!";
```

Is this a grammatical error?